website/src/models/group.js

```diff
@@ -87,23 +87,46 @@ GroupSchema.pre('save', function(next){
 
 GroupSchema.pre('remove', function(next) {
   var group = this;
-  async.waterfall([
-    function(cb) {
-      User.find({
-        'invitations.guilds.id': group._id
```

Comment:
I can try, but I believe the invitations are structured differently. The party invitations weren't accepting the same type of array queries as the guilds were.